### PR TITLE
Join className from container with the one from child element

### PR DESCRIPTION
**Pull request by v12**
Fixes https://github.com/react-bootstrap/react-router-bootstrap/issues/218

### CODE REVIEW

File: src/LinkContainer.js [hunk 1]
```diff
@@ -97,10 +99,10 @@ export default class LinkContainer extends Component {
           const isActive = !!(getIsActive ? getIsActive(match, location) : match);
 
           return React.cloneElement(
-            React.Children.only(children),
+            child,
             {
               ...props,
-              className: isActive ? [className, activeClassName].join(' ') : className,
+              className: [className, child.props.className, isActive && activeClassName].join(' '),
```
Comment (taion) [inline src/LinkContainer.js]: This isn't right:

```js
> ['asdf', false].join(' ')
'asdf false'
```

Maybe just pull in `classnames`?

Comment (v12) [reply src/LinkContainer.js]: Ah, it was supposed to be `null` - missed that, thanks.
Do you reckon it's worth to have another dependency just to join three different classes? 🤔

Comment (taion) [reply src/LinkContainer.js]: Huh, I thought it was a dep in the previous version. I guess it never came up?

I mean, React-Bootstrap pulls in classnames anyway, so it's not an extra dependency net for anyone already using React-Bootstrap. Or really classnames is just so darn common, no?

Comment (v12) [reply src/LinkContainer.js]: `classnames` is quite common for many React project, although, this module isn't restricted to be used with react-bootstrap only – i.e., I started to use it with [Blueprint](http://blueprintjs.com/) recently and it works like a charm. 😃 

I'd keep it as tiny as possible as I don't see an advantage of bringing additional dependency for such a simple use case 😉

Comment (taion) [reply src/LinkContainer.js]: Blueprint uses classnames too:

https://github.com/palantir/blueprint/search?utf8=%E2%9C%93&q=classnames&type=

Hand-rolling the class joining would be the worst thing. I'd personally use `classnames` but it's your call – either way is fine.

Comment (taion) [reply src/LinkContainer.js]: GAH sorry typed too quickly, I meant _wouldn't_ be the worst thing. sorry ):

Comment (v12) [reply src/LinkContainer.js]: It does, you're right. 👍

I mentioned Blueprint just as one of the examples that react-router-bootstrap can be used with other libraries (not even libraries but, potentially, custom components).

Anyway, if this comes up at any point later, I'll pull `classnames` in 😉 

Thanks for an instant response and valuable comments!

**Pull request merged**